.travis.yml

```diff
@@ -1,20 +1,40 @@
 language: java
-
-# Skip testing for now, since this is experimental
-script: ./gradlew build -x test
-
 jdk:
   - oraclejdk8
 
+# Anything after trusty doesn't have Java 8
 dist: trusty
 
 # Below skips the installation step completely (https://docs.travis-ci.com/user/customizing-the-build/#Skipping-the-Installation-Step)
-# If we don't skip it, Travis runs unnecessary Gradle tasks like './gradlew assemble'
-install:
-  - true
+# If we don't skip it Travis runs unnecessary Gradle tasks like './gradlew assemble'
+install: true
 
-# Only build on master and release tags
+# Build and test commits to master, PRs to master, and release tags
+# Skip testing for now, since this is experimental
+script: ./gradlew build -x test
 branches:
   only:
-  - master
-  - /^v\d+\.\d+\.\d+$/
+    - master
+    - /^v\d+\.\d+\.\d+$/
+
+# Set environment variables (includes Bintray credentials)
+env:
+  global:
+    - secure: 'DJbT0+O3W57jxaXmpVvuY4AH5VDqAgvmAwtw9GLK7fHhntmKMh8o43xWU7Tmv5LUPDU/UChXzte5fRPSe75KvkS72Rie02M/MSizO9B9fWGQ5trzWqCqGO4mI38v9e2PfInOWa7R/FA3z1NYas11CulkTXQI+a10po7nzIMrEnM='
+    - secure: 'b4Q5mdqAQJcyuDyvxZK6MiImxho18HteKMDT0UANpIyrFXEfzWKIlB110unofXSXKiI0bAg+ob/PLRULkB/wuj69WasXRiH83+Ao2WpNbWyMoqPwiY/DL8MN0M1RhHbuSMso/O6C34eRImhj5NDcN+nkI+py8pHAq9FNrmQ92O4='
```

Comment:
can you add super_secret, plain text credentials to our internal wiki? for hotfix releases and bookkeeping
also, are these required here? can we not add these in repository settings?

Reply:
I won't add these as plaintext, because these are my personal credentials. I'll add instructions for how to set them yourself or how to replace them.
As for configuring them via repo settings, I'd have to do it in Travis settings rather than GitHub settings (I don't think they can be accessed here). However, I don't have Travis admin permissions for LinkedIn. I see other repos setting it like this, so it seems to be a common practice.

Reply:
is it possible possible to create the account using our team email id? or can we have 1 common account?

Reply:
No, I asked Tools and the only way to authenticate as of now is to use personal credentials.